library/src/compute/foundations.rs

```diff
@@ -115,6 +123,90 @@ pub fn assert(
     Value::None
 }
 
+/// Ensure that two values are equal.
+///
+/// Fails with an error if the first value (expected) is not equal to the
+/// second (actual). Does not produce any output in the document.
+///
+/// ## Example
+/// ```example
+/// #assert.eq(10, 10)
+/// ```
+///
+/// Display: Assert Equals
+/// Category: foundations
+/// Returns:
+#[func]
+pub fn assert_eq(
+    /// The expected value.
+    expected: Value,
+
+    /// The actual value.
+    actual: Value,
+
+    /// An optional message to display on error
+    /// instead of the representations of the compared values.
+    #[named]
+    #[default]
+    message: Option<EcoString>,
+) -> Value {
+    if !typst::eval::ops::equal(&expected, &actual) {
```

Comment:
`!typst::eval::ops::equal(&expected, &actual)` is the same as `expected != actual`, see [here](https://github.com/typst/typst/blob/b5d72caaf94a5079cf67ce5c4764992d5e9ed01a/src/eval/value.rs#L201)

Reply:
Oooh, I missed that, thanks.